tests/lsdb/catalog/test_crossmatch.py

```diff
@@ -124,6 +124,7 @@ def test_right_margin_missing(algo, small_sky_catalog, small_sky_xmatch_catalog)
             small_sky_catalog.crossmatch(small_sky_xmatch_catalog, algorithm=algo, require_right_margin=True)
 
 
+@pytest.mark.skip(reason="HIPSCAT2HATS")
```

Comment:
😎 I didn't know about this decorator. Nice!

Reply:
Nice, right? Makes it pretty easy to find the cases we know are broken for now, and have CI still work for the ones that SHOULD work.